kitsune/sumo/static/sumo/less/main.less

```diff
@@ -1229,6 +1229,30 @@ input[type=button],
     position: relative;
     top: -2px;
   }
+
+  .firefox-download-button {
+    background: #81BC2E;
+    background: linear-gradient(to bottom, #81BC2E, #659324);
+    border-radius: 5px;
+    box-shadow: 0 1px 0 0 rgba(0, 0, 0, 0.2), 0 -1px 0 0 rgba(0, 0, 0, 0.3) inset;
+    display: inline-block;
+    line-height: 25px;
+    margin: 6px 15px 0px 15px;
+    position: relative;
+    text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.25);
+    transition: top 0s;
+    transition: all 0.5s;
```

Comment:
This combination of rules is redundant. Later rules always overwrite previous rules, so the `transition: top 0s;` rule is erased. If you want this to be "all except top", then you should swap the order of the rules. Otherwise, just remove the first transition rule.